app/helpers/application_helper.rb

```diff
@@ -159,6 +159,10 @@ def logo_svg
     end
   end
 
+  def show_logo logo
```

Comment:
We should start using `show_logo` helper to display what logo to use. We're going to find that many Forems will most likely have only one logo, so in the case where additional logos/images are not available (like a secondary_logo, onboarding_logo_image and onboarding_taskcard_image), we will use the primary logo image in its place 😊 Hence, this allows a user to get started with Forem more easily and allows us to reduce the number of madatory configs

Reply:
Total nitpick, but we don't generally use the no parenthesis form for method defintions.

Reply:
Ah, sorry for that, sometimes it gets confusing between the conversation s and the actual diffs, especially when you don't notice the "Outdated" tag.